crates/sui-indexer-alt-graphql/src/api/types/event/filter.rs

```diff
@@ -20,11 +31,211 @@ pub(crate) struct EventFilter {
 
     /// Limit to event that occured strictly before the given checkpoint.
     pub before_checkpoint: Option<UInt53>,
-    // TODO: (henry) Implement these filters.
-    // pub sender: Option<SuiAddress>,
-    // pub transaction_digest: Option<Digest>,
-    // pub module: Option<ModuleFilter>,
-    // pub type: Option<TypeFilter>,
+
+    /// Filter on events by transaction digest.
+    pub digest: Option<Digest>,
```

Comment:
We should probably call this `transaction_digest` to avoid confusion -- events have digests as well on Sui (as do effects, which are events are part of).

Reply:
Ah right, ty!